test/base.py

```diff
@@ -45,33 +48,36 @@ def flatten(l, ltypes=(list, tuple)):
         i += 1
     return ltype(l)
 
+
 class ColumnName(str):
     pass
 
+
 def dbparam(x):
     if x is None:
         return 'NULL'
     elif isinstance(x, ColumnName):
         return "`" + x + "`"
-    elif isinstance(x, basestring):
+    elif isinstance(x, bytes):
```

Comment:
Correct. Unicode has to be handled in the next clause though if you want bytes to work properly on python3.